javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/EarlyInitAgentConfig.java

```diff
@@ -14,9 +14,8 @@
  */
 public final class EarlyInitAgentConfig {
 
-  public static EarlyInitAgentConfig create() {
-    return new EarlyInitAgentConfig(ConfigurationFileLoader.getConfigFileContents());
-  }
+  public static final EarlyInitAgentConfig INSTANCE =
+      new EarlyInitAgentConfig(ConfigurationFileLoader.getConfigFileContents());
```

Comment:
This actually introduces a chicken-and-egg problem: `ConfigurationFileLoader` uses loggers, and these cannot be used before the logging system is initialized; which means you cannot use `EarlyInitAgentConfig` before `LoggingCustomizer` runs, the way the code is shaped right now.
@0x006EA1E5 I'll try to come up with a few ideas on how to refactor or circumvent that, will come back to this PR in a couple days.